honeybee_radiance/cli/grid.py

```diff
@@ -43,7 +44,8 @@ def split_grid(grid_file, count, folder, log_file):
         grid = sensorgrid.SensorGrid.from_file(grid_file)
         file_count = int(round(grid.count / count))
         files = grid.to_files(folder, file_count, mkdir=True)
-        log_file.write('\n'.join(files))
+        
+        log_file.write(json.dumps(files))
```

Comment:
Curios to know why do we need json.dumps in this case?

Reply:
It turns out that if you load a file as an output parameter in Argo and you then want to use that parameter to loop tasks, then the parameter must be a JSON serializable array.

ie:
## BAD
```
room_0001.pts \n
room_0002.pts 
``` 

## GOOD
```
[ "room_0001.pts", "room_0002.pts" ]
```
```
[
  {
    "name": "room_0001",
    "path": "room_0001.pts"
  },
  {
    "name": "room_0002",
    "path": "room_0002.pts"
  }
]
```

Reply:
Got it! Thanks. This might be a better practice anyways.